src/globals.js

```diff
@@ -275,23 +275,27 @@ for (const key in geoKeyNames) {
 }
 
 export let parseXml; // eslint-disable-line
+export let UNORDERED_NODE_SNAPSHOT_TYPE; // eslint-disable-line
 // node.js version
 if (typeof window === 'undefined') {
   parseXml = function (xmlStr) {
     // requires xmldom module
     const { DOMParser } = require('xmldom'); // eslint-disable-line global-require
     return (new DOMParser()).parseFromString(xmlStr, 'text/xml');
   };
+  UNORDERED_NODE_SNAPSHOT_TYPE = require('xpath').XPathResult.UNORDERED_NODE_SNAPSHOT_TYPE; // eslint-disable-line
```

Comment:
Imports should be on the top of the file. I know this is difficult in this case, because of the "externals" definition, but having the import down here is very confusing.

Reply:
removed export of xpath in globals.js because it is no longer required here (still used in geotiffimage.js)